Student_BCOs/(FinalUpdate)Juntian_Wei_BCO_000549.json

```diff
@@ -0,0 +1,434 @@
+            {
+                "name": "No dependencies",
+                "version": "",
+                "step_number": 4,
+                "input_list": [],
+                "output_list": [],
+                "prerequisite": [],
+                "description": "The final step has no dependencies. TargetScan and miRanda are used, which are two different websites. TargetScan: http://www.targetscan.org/vert_72/. miRanda: http://www.microrna.org/microrna/home.do"
+            }
+        ]
+    },
+    "execution_domain": {
+        "external_data_endpoints": [
+            {
+                "name": "MiRanda",
+                "url": "http://www.microrna.org/microrna/home.do"
+            },
+            {
+                "name": "miRanalyzer subsection 3.1",
+                "url": "https://bioinfo2.ugr.es/miRanalyzer/standalone.html#toc-Subsection-3.1"
+            }
+        ],
+        "environment_variables": {},
+        "script_driver": "",
+        "software_prerequisites": [
+            {
+                "name": "Python",
+                "version": "2.7 or higher",
+                "uri": {
+                    "access_time": "",
+                    "uri": "https://www.python.org/getit/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "PERL",
+                "version": "5.0 or higher",
+                "uri": {
+                    "access_time": "",
+                    "uri": "http://www.perl.org/get.html",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "Java",
+                "version": "1.6 or higher",
+                "uri": {
+                    "access_time": "",
+                    "uri": "http://www.java.com/en/download/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "R",
+                "version": "3.0.1 or higher",
+                "uri": {
+                    "access_time": "",
+                    "uri": "http://mirror.fcaglp.unlp.edu.ar/CRAN/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "gcc",
+                "version": "9.2",
+                "uri": {
+                    "access_time": "",
+                    "uri": "https://gcc.gnu.org/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "make",
+                "version": "4.2",
+                "uri": {
+                    "access_time": "",
+                    "uri": "http://ftp.gnu.org/gnu/make/",
+                    "sha1_checksum": "000528checksum",
```

Comment:
The ID for the submitted BCO is BCO_000549. 
Please modify this checksum accordingly.